### DISCUSSION TITLE
Easy domain name change/switch for requests

### DISCUSSION THREAD

**Discussion by xiki808** [Kong/insomnia#4426] [Ideas] [answered]
I think it would be quite convenient to have an easy way to switch the domain part of a request. 

For example, just two clicks to change 

from : http://localhost:8888/backup/create) 
to     : http://qa.anotherdomain.com/backup/create

Normally the project I am working on is hosted in multiple instances, locally, on a dev/qa/staging/production instance on the cloud.

**Comment by dimitropoulos** [comment, marked as answer]
would it work if you used environment variables for this?  you can switch between them with a keyboard shortcut fairly easily.

![Screenshot_20220201_170643](https://user-images.githubusercontent.com/15232461/152059465-e4d84789-07ef-4193-9d6f-e67f0f89b76d.png)

![Screenshot_20220201_170658](https://user-images.githubusercontent.com/15232461/152059485-79ee2474-18da-4d5f-8cf6-e8943adfd926.png)

![Screenshot_20220201_170631](https://user-images.githubusercontent.com/15232461/152059444-88c4b723-f178-4ff5-b908-fee2a2ee2b0e.png)

![Screenshot_20220201_170724](https://user-images.githubusercontent.com/15232461/152059543-cf882072-aed0-44d0-b8d8-001c0b38ef11.png)

**Reply by xiki808** [reply]
Yes this is something I was looking for, had no idea about environments, sorry for that! Thank you for your answer.
